packages/audioplayers/lib/src/audioplayers_web.dart

```diff
@@ -44,6 +49,9 @@ class WrappedPlayer {
     player?.loop = shouldLoop();
     player?.volume = currentVolume;
     player?.playbackRate = currentPlaybackRate;
+    playerTimeUpdateSubscription = player?.onTimeUpdate.listen((event) =>
+        plugin.channel.invokeMethod('audio.onCurrentPosition',
+            {'value': (1000 * (player?.currentTime ?? 0)).round()}));
```

Comment:
wrong formatting here

```suggestion
        plugin.channel.invokeMethod(
            'audio.onCurrentPosition',
            {'value': (1000 * (player?.currentTime ?? 0)).round()},
        ),
    );
```